RM_Vk/RM_Vk/Model/User.swift

```diff
@@ -0,0 +1,168 @@
+// User.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Генерация юзеров
```

Comment:
```suggestion
/// Генерация юзеров
```
а называется юзер фактори)